using_images/other_images/jenkins.adoc

```diff
@@ -293,6 +293,16 @@ The default maven agent pod uses `docker.io/openshift/jenkins-agent-maven-35-cen
 the CentOS or RHEL version of the Jenkins image. This variable must be set before Jenkins starts
 the first time for it to have an effect.
 
+* `JENKINS_UC_INSECURE`
++
+Determines whether Jenkins plugins downloads are allowed if the Jenkins Update Center repository 
+uses an invalid SSL certificate. This could be the case if a self hosted repository 
+using self-signed certificate with an unknow CA is used or if an enteprise proxy  
+performs man-in-the-middle interceptions. This variable applies to plugins downloads which may 
```

Comment:
I would suggest changing this sentence: "This variable applies to plugins downloads which may occur during Jenkins image build or if an extension of the Jenkins image is built or if you run the Jenkins image and leverage one of the options to download additional plugins (use of s2i whith plugins.txt or use of INSTALL_PLUGINS environment variable). To enable, set to true." 

To: 

"This variable applies to plug-in downloads, which may occur during a Jenkins image build or if an extension of the Jenkins image is built. It is also applied when you run the Jenkins image and use one of the options to download additional plug-ins, including S2I with `plugins.txt` or the `INSTALL_PLUGINS` environment variable. Set to true to enable this variable."